cvmfs/swissknife_lease.cc

```diff
@@ -0,0 +1,141 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "swissknife_lease.h"
+
+#include <fstream>
```

Comment:
I can't really pinpoint a particular problem.  The Google style guide has a couple of cons (https://google.github.io/styleguide/cppguide.html#Streams).  Not using streams mostly means one less thing we need to know and care about.